test/integration/test_holidays.rb

```diff
@@ -293,6 +293,6 @@ def test_caching
 
   def test_load_all
     Holidays.load_all
-    assert_equal 295, Holidays.available_regions.count
+    assert_equal 170, Holidays.available_regions.count
```

Comment:
This is actually a bit embarrassing. I apparently had it putting duplicates in the region count that was called. After my fix I found that my number was MUCH lower, but the new number is actually the correct one. :feelsgood: